### PR TITLE
#35 Write an API to set password using email link

### PR DESCRIPTION
**Pull request by aashishpiitk**
#43 Write an API to reset password using email link

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -59,12 +64,135 @@ def ActivationMailer(request):
...
+    if request.method == "POST":
+        try:
+            new1=request.data['new_password1']
+            new2=request.data['new_password2']
+            old=request.data['old_password']
+            user_data=User.objects.get(verification_code=token)
+            password=(user_data.password)
+            password=password.encode()
+
+            print(pass_checker(old,password))
```
Comment (vikrantmalik051) [inline users/views.py]: Please remove this `print` statement if it was used while debugging.

File: users/jobs/daily/scrape.py [hunk 2]
```diff
@@ -106,7 +106,7 @@ def execute(self,soup,c):
             match = pattern.match(substituted)
             TOTAL = int(match.group(1))
             print("Total: {}".format(TOTAL))
-            conn.close()
+            #conn.close()
```
Comment (vikrantmalik051) [inline users/jobs/daily/scrape.py:109]: Is this done deliberately? I guess it should be closed.

Comment (aashishpiitk) [reply users/jobs/daily/scrape.py:109]: no it shouldn't be closed
if uncommented then it is useful in checking the total number of users without adding them to the database

Comment (aashishpiitk) [reply users/jobs/daily/scrape.py:109]: should i remove the commet 
but it would be helpful in future

File: users/views.py (hunk 1 above)
Comment (aashishpiitk) [reply users/views.py]: i will do it

File: users/jobs/daily/scrape.py (hunk 2 above)
Comment (vikrantmalik051) [reply users/jobs/daily/scrape.py:109]: The total number would be fixed for now. So it should be closed i.e., it should be uncommented.

Comment (aashishpiitk) [reply users/jobs/daily/scrape.py:109]: magar agar close kar dunga toh fir daily job ka kya fayada
fir daily us script ke run karne pe the databse won't be updated

**Pull request merged**